optimum/commands/export/onnx.py

```diff
@@ -100,6 +100,19 @@ def parse_args_onnx(parser):
             " and decoder-with-past models into a single ONNX model file to reduce memory usage."
         ),
     )
+    optional_group.add_argument(
+        "--optimize",
+        type=str,
+        default=None,
+        choices=["O1", "O2", "O3", "O4"],
```

Comment:
So no possibility of providing an `ORTConfig`?

Reply:
I will in a next PR.